src/invocation/ClusterService.ts

```diff
@@ -212,25 +226,26 @@ export class ClusterService implements Cluster {
         return (listener as InitialMembershipListener).init !== undefined;
     }
 
-    private applyInitialState(memberListVersion: number, memberInfos: MemberInfo[]): Promise<void> {
-        const snapshot = ClusterService.createSnapshot(memberListVersion, memberInfos);
+    private applyInitialState(memberListVersion: number, memberInfos: MemberInfo[], clusterUuid: UUID): Promise<void> {
+        const snapshot = ClusterService.createSnapshot(memberListVersion, memberInfos, clusterUuid);
         this.memberListSnapshot = snapshot;
         this.logger.info('ClusterService', ClusterService.membersString(snapshot));
         const members = snapshot.memberList;
-        const event = new InitialMembershipEvent(members);
+        const event = new InitialMembershipEvent(this, members);
         this.listeners.forEach((listener) => {
             if (ClusterService.isInitialMembershipListener(listener)) {
                 listener.init(event);
             }
         });
         const addressProvider =
-            this.clusterFailoverService.current().addressProvider;
+        this.clusterFailoverService.current().addressProvider;
         return this.translateToAddressProvider.refresh(addressProvider, memberInfos);
     }
 
     private detectMembershipEvents(
         prevMembers: MemberImpl[],
         currentMembers: MemberImpl[],
+        clusterUuid: UUID,
         connectionRegistry: ConnectionRegistry
```

Comment:
this parameter is unused, can you remove this?

Reply:
Fixed